src/transport.ts

```diff
@@ -112,7 +99,8 @@ export class WebRTCTransport implements Transport, Initializable {
     //set the Noise Prologue to libp2p-webrtc-noise:<FINGERPRINTS> before starting the actual Noise handshake.
     //  <FINGERPRINTS> is the concatenation of the of the two TLS fingerprints of A and B in their multihash byte representation, sorted in ascending order.
     let fingerprintsPrologue = this.generateNoisePrologue(peerConnection, ma);
-    let noise = new Noise(myPeerId.privateKey, undefined, stablelib, fingerprintsPrologue);
+    let noise = new Noise(undefined, undefined, undefined, fingerprintsPrologue);
```

Comment:
Are these undefines really what we want here? Also if that is an expected use case than IMHO that function signature is pretty poor, as you would in JS expect it to be part as an optional dict at the end instead, forcing to put these undefined is a pretty poor choice.

Or is this just a temporary thing on our end? either way, might at the very least want to document why you're putting undefines here

Reply:
This is the function signature expected by the Noise library: https://github.com/ChainSafe/js-libp2p-noise/blob/master/src/noise.ts#L38
We do not use any of the earlier parameters and it does not accept a dict, so we are forced to put these undefined. We can always make a PR to the Noise library